filesystem/Upload.php

```diff
@@ -175,11 +186,12 @@ public function load($tmpFile, $folderPath = false) {
 				$i = isset($i) ? ($i+1) : 2;
 				$oldFilePath = $relativeFilePath;
 
-				$pattern = '/([0-9]+$)/';
-				if(preg_match($pattern, $fileTitle)) {
-					$fileTitle = preg_replace($pattern, $i, $fileTitle);
+				$prefix = $this->config()->version_prefix;
```

Comment:
Be careful if the prefix contains regex characters. E.g. `'.ver'`. Maybe preg_quote this first?